source/about.html.erb

```diff
@@ -1,3 +1,44 @@
 ---
 title: Jeff Stolz | About
----
\ No newline at end of file
+---
+
+<%= image_tag "profile.png", :class => "about__profile", :alt => t("about.profile-alt") %>
```

Comment:
Looks like `profile.png` is 5.03 MB. It’d be great to optimize down to a much smaller file size!

Reply:
Agreed. And if you can switch the file type to `jpg` (unless you require some transparency for the `png`), that would be even better since those files tend to be smaller.